tests/e2e/specs/in-game/basicMoves.spec.js

```diff
@@ -57,6 +63,17 @@ describe('Game Basic Moves - P0 Perspective', () => {
       p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
       p1FaceCards: [Card.KING_OF_HEARTS],
     });
+
+    cy.wait(500).then(() => {
```

Comment:
It's actually not the loaded app that I was waiting for, but rather just the `console` messages to flush since console writes are asynchronous and I wasn't getting any console messages without this.

I'll remove this though